### PR TITLE
Fix missing builds with no build config

### PR DESCRIPTION
**Pull request by jwforres**

### CODE REVIEW

File: app/views/builds.html [hunk 1]
```diff
@@ -135,6 +131,51 @@
                       </span>
                     </td>
                   </tr>
+
+                  <!-- Builds that were not spawned from a build config -->
+                  <tr ng-repeat="latestBuild in buildsNoConfig">
```
Comment (spadgett) [inline app/views/builds.html]: I know this was copied from above, but should use `track by (latestBuild | uid)`

**Comment by jwforres** [comment]
@spadgett updated track by on both ng-repeats

File: app/views/builds.html [hunk 2]
```diff
@@ -49,11 +49,11 @@
...
                   </tr>
                 </thead>
-                <tbody ng-if="!(latestByConfig | hashSize)">
+                <tbody ng-if="!(latestByConfig | hashSize) && !(buildsNoConfig | hashSize)">
                   <tr><td colspan="7"><em>{{emptyMessage}}</em></td></tr>
                 </tbody>
-                <tbody ng-if="(latestByConfig | hashSize)">
-                  <tr ng-repeat="(buildConfigName, latestBuild) in latestByConfig">
+                <tbody ng-if="(latestByConfig | hashSize) || (buildsNoConfig | hashSize)">
+                  <tr ng-repeat="(buildConfigName, latestBuild) in latestByConfig track by (latestBuild | uid)" ng-if="buildConfigName !== ''">
```
Comment (spadgett) [inline app/views/builds.html]: Nit: might be better as `track by buildConfigName` since the row is for the BC?

Comment (jwforres) [inline app/views/builds.html]: hmm good point, do we actually even need track by in this case then? will ng-repeat track by the key for a hash? looking...

**Comment by jwforres** [comment]
fixed [merge][severity:blocker]

**Pull request merged**